bay/plugins/run.py

```diff
@@ -77,25 +80,50 @@ def run(app, containers, host, tail):
 @click.command()
 @click.argument("container", type=ContainerType())
 @click.option("--host", "-h", type=HostType(), default="default")
+@click.option("--running", "-r", is_flag=True)
+@click.option("--not-running", "-n", is_flag=True)
 @click.argument("command", nargs=-1, default=None)
 @click.pass_obj
-def shell(app, container, host, command):
+def shell(app, container, host, running, not_running, command):
```

Comment:
Is there some way of detecting if it's been called via `shell` or `attach`? IMO in case of being called via `shell` it should default to `not_running=True` to preserve the way it works now.